modules/installing-ocp-agent-manifests.adoc

```diff
@@ -0,0 +1,162 @@
+// Module included in the following assemblies:
+//
+// * installing-with-agent/installing-with-agent-based-installer.adoc
+
+:_content-type: PROCEDURE
+[id="installing-ocp-agent-manifests_{context}"]
+= Creating the preferred manifest files
+
+Use this procedure to create and configure the preferred manifest files used to create the agent image.
```

Comment:
The install-config and agent-config aren't manifest files as such; they don't get installed into the kubernetes API as a CR.